hecuba_core/tests/runtests.cpp

```diff
@@ -2515,14 +2512,630 @@ TEST(TestingCacheTable, DeleteRow) {
 
     EXPECT_TRUE(results.empty());
 
-    delete(keys);
-    delete(read_values);
-    delete(cache);
+    delete (keys);
+    delete (read_values);
+    delete (cache);
 
     CassFuture *close_future = cass_session_close(test_session);
     cass_future_wait(close_future);
     cass_future_free(close_future);
 
     cass_cluster_free(test_cluster);
     cass_session_free(test_session);
+}
+
+#include <iostream>
+#include "gtest/gtest.h"
```

Comment:
Includes go to the top of the file. Besides, I guess that they are already in-place